linera-rpc/tests/snapshots/format__format.yaml.snap

```diff
@@ -639,12 +637,10 @@ OracleResponse:
   ENUM:
     0:
       Service:
-        NEWTYPE:
-          SEQ: U8
+        NEWTYPE: BYTES
     1:
       Post:
-        NEWTYPE:
-          SEQ: U8
+        NEWTYPE: BYTES
```

Comment:
bit surprised that this PR changed the format.yaml file as well.

Reply:
Yes, that's because of the `serde_bytes`. I could split these changes out, to separate them from the `Debug` ones.